x-pack/plugins/security_solution/common/utils/path_placeholder.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConditionEntryField, OperatingSystem, TrustedAppEntryTypes } from '../endpoint/types';
+
+export const placeholderText = {
```

Comment:
I suggest that this be a function instead of object definition in order to ensure they are not mutated.
Also - are these for testing? if so, I would also suggest moving them to a file (or directory) whose name as "test" in it. like `test_utils.ts`

Reply:
Ah, I see your point with the mutation. Although I'm just reusing the same text values within the function that shows the placeholder text and also within the unit tests so I think moving it to a `test_utils.ts` file is not necessary.